uber/templates/preregistration/preregbase.html

```diff
@@ -64,12 +66,12 @@
             row: '#badge-types',
             selector: '.badge-type-selector',
             options: [{
```

Comment:
Well I deliberately ONLY wanted it to say "Staffer" for people who actually have a staff badge, as opposed to just people with a volunteer ribbon, so I don't think ``ribbon_and_or_badge`` would be appropriate here.